packages/@aws-cdk/aws-sagemaker-alpha/lib/model.ts

```diff
@@ -206,6 +206,17 @@ export interface ModelProps {
    * @default true
    */
   readonly allowAllOutbound?: boolean;
+
+  /**
+   * Whether to isolate the model container.
```

Comment:
I think this could be more descriptive. Maybe combine this with the sentence below.

Reply:
I have combined the sentences below. If you have any better suggestions, please let me know!